system/Database/MySQLi/Forge.php

```diff
@@ -187,11 +187,18 @@ protected function _processColumn(array $field): string
      *
      * @param string $table (ignored)
      */
```

Comment:
Added second parameter $asQuery. This tells the function to return a stand alone query rather than the partial sql used when creating a table.